components/atoms/Radio/radio.tsx

```diff
@@ -16,6 +16,14 @@ const Radio = ({ className, withLabel, id, children, value, checked = false, onC
   return (
     <div
```

Comment:
This should be an actual button, and we can remove the `tabIndex` and `role="button"`